components/cam/bld/configure

```diff
@@ -980,24 +973,6 @@ my $clubb_do_adv = $cfg_ref->get('clubb_do_adv');
 if ($print>=2) { print "clubb_do_deep=',$clubb_do_deep,$eol"; }
 if ($print>=2) { print "clubb_do_adv=',$clubb_do_adv,$eol"; }
 
-#-----------------------------------------------------------------------------------------------
```

Comment:
just noting locations where shoc/p3 might be removed.

Reply:
we need to retain all lines/code blocks that have P3